contrib/sarplus/scala/python/setup.py

```diff
@@ -1,8 +1,13 @@
 from distutils.core import setup
+import os
+
+version = os.getenv("VERSION")
+if version is None:
+    version = "0.5.0"
 
 setup(
     name="pysarplus_dummy",
-    version="0.2",
+    version=os.environ["VERSION"],
     description="pysarplus dummy package to trigger spark packaging",
     author="Markus Cozowicz",
     author_email="marcozo@microsoft.com",
```

Comment:
change url to https://github.com/microsoft/recommenders/tree/main/contrib/sarplus

Reply:
done